deep_quoridor/src/agents/alphazero/mcts.py

```diff
@@ -150,7 +149,7 @@ def select(self, node: Node) -> Node:
 
     def search(self, initial_game: Quoridor):
         root = Node(initial_game, ucb_c=self.ucb_c)
-        self.new_nodes = []
+        new_nodes = []
```

Comment:
This is unrelated, just piggy-backing a comment by @jonbinney in a previous PR about this var being local